packages/integrations/tailwind/package.json

```diff
@@ -28,20 +28,19 @@
     "dev": "astro-scripts dev \"src/**/*.ts\""
   },
   "dependencies": {
-    "@proload/core": "^0.3.2",
     "autoprefixer": "^10.4.7",
     "postcss": "^8.4.14",
     "postcss-load-config": "^4.0.1"
   },
   "devDependencies": {
     "astro": "workspace:*",
     "astro-scripts": "workspace:*",
-    "tailwindcss": "^3.0.24",
+    "tailwindcss": "3.3.0",
```

Comment:
```suggestion
    "tailwindcss": "^3.3.0",
```

Reply:
what was the reason for not pinning this?

Reply:
makes sense thanks for clarifying.